src/instance.rs

```diff
@@ -412,8 +141,8 @@ impl<T> IntoPy<PyObject> for Py<T> {
     /// Converts `Py` instance -> PyObject.
     /// Consumes `self` without calling `Py_DECREF()`
     #[inline]
-    fn into_py(self, py: Python) -> PyObject {
-        unsafe { PyObject::from_owned_ptr(py, self.into_ptr()) }
+    fn into_py(self, _py: Python) -> PyObject {
+        unsafe { PyObject::from_not_null(self.into_non_null()) }
```

Comment:
Naming is inconsistent: `not_null` and `non_null`. Maybe one can be tweaked so they're consistent?

Reply:
Yeah, we should fix this.